### ISSUE TITLE
Doc the units when giving size from analysis

### ISSUE THREAD

**Issue by ben-spiller** [newren/git-filter-repo#251]
It's not 100% clear whether all the sizes reported by the tool are in bytes or kB. 

Maybe the generated analysis output text files could include units e.g. 
=== All extensions by reverse size ===
- Format: unpacked size, packed size, date deleted, extension name
+ Format: unpacked size bytes, packed size bytes, date deleted, extension name

**Comment by newren** [comment]
Yeah, all sizes are in bytes; fixing up the docs would make sense.
